client/app/common/dashboard/service/no-hubic.html

```diff
@@ -1,4 +1,4 @@
 <div class="top-space-m16">
```

Comment:
`top-space-m16` => `mt-3`

Reply:
This is part one other css class will come along...